app/views/welcome/index.html.erb

```diff
@@ -1,8 +1,25 @@
 <span><p><%= @quote %></p></span>
 <main role="main" class="content">
+
+  <% if @is_admin_approved%>
+  <div class="alert alert-warning" role="alert">
+  Thank you for signing up! You are now a mentor!
+  </div>
+
+  <% elsif @is_manager_approved%>
+  <div class="alert alert-warning" role="alert">
+  Thank you for signing up! Pending admin approval.
+  </div>
+
+  <% elsif @is_mentor %>
+  <div class="alert alert-warning" role="alert">
+  Thank you for signing up! Pending manager approval.
+  </div>
+  <% end %>
+
```

Comment:
Nice. Another way to do this would be to return the message you want to show from the method call, that way if you change the styling you only have to change it in one place.

Reply:
Thank you Muffy! That's a good point - I am also thinking of hiding the application form once someone has signed up. I'll definitely create a method call for that.

@xavierwalker I can move status changes into a method as well - would that be more convenient for the layouts you're working on?